systems/plants/TimeSteppingRigidBodyManipulator.m

```diff
@@ -17,6 +17,7 @@
     enable_fastqp; % whether we use the active set LCP
     lcmgl_contact_forces_scale = 0;  % <=0 implies no lcmgl
     z_inactive_guess_tol = .01;
+    mex_model_ptr = 0; % needed for access from mex functions
```

Comment:
is it?  can't you get property manip, then get property mex_model_ptr?  is that too expensive?